ssc/cmod_battery.cpp

```diff
@@ -2160,11 +2187,11 @@ class cm_battery : public compute_module
 
                         if (resilience) {
                             resilience->add_battery_at_outage_timestep(*batt->dispatch_model, lifetime_idx);
-                            resilience->run_surviving_batteries(p_crit_load[lifetime_idx % n_rec_single_year], power_input_lifetime[lifetime_idx]);
+                            resilience->run_surviving_batteries(p_crit_load_full[lifetime_idx], power_input_lifetime[lifetime_idx]);
```

Comment:
Is p_crit_load_full the percentage (or timeseries) entered by the user?

Reply:
`p_crit_load_full` is a timeseries extended to lifetime by the compute module. `p_crit_load` is the year 1 timeseries. The GUI converts percentages into a kW timeseries in the "$MIMO$ elecload" equations on the Electric Load Crit page.